MDANSE/Src/MDANSE/MolecularDynamics/Trajectory.py

```diff
@@ -561,62 +601,69 @@ def write_atom_properties(
             if properties["element"] == "dummy":
                 properties["dummy"] = 1
         new_labels = [str(x) for x in properties.keys()]
```

Comment:
Are property keys ever not a string?
```suggestion
        new_labels = list(properties)
```